### ISSUE TITLE
Antígenos

### ISSUE THREAD

**Issue by adinamarca** [MinCiencia/Datos-COVID19#994]
¡Hola!

En el [producto 3](https://github.com/MinCiencia/Datos-COVID19/blob/master/output/producto3/TotalesPorRegion.csv) es posible ver que, hace un tiempo, se están reportando los **casos confirmados por antigeno** y **casos nuevos confirmados por antígeno** (y ésto fue paralelo a la añadidura de los datos al reporte diario del Minsal).

Respecto a esta "añadidura", observé que se añadió mayor detalle a los **antígenos** ([en la página 13 del reporte diario del Minsal](https://s3.amazonaws.com/gobcl-prod/public_files/Campa%C3%B1as/Corona-Virus/Reportes/07.06.2021_Reporte_Covid19.pdf)). En concreto:

- Exámenes de antígeno informados de acuerdo al último informe epidemiológico.

¿Es posible contar con este número en la base de datos COVID-19 del Min. Ciencia? Y en caso de ser posible, ¿disponibilidad diaria?

Desde ya, muchas gracias por su enorme trabajo.

¡Saludos!

**Comment by dna33** [comment]
Hola Alejandro, lo agregamos el fds, está acá:

https://github.com/MinCiencia/Datos-COVID19/tree/master/output/producto87

**Comment by adinamarca** [comment]
> Hola Alejandro, lo agregamos el fds, está acá:
> 
> https://github.com/MinCiencia/Datos-COVID19/tree/master/output/producto87

¡Hola Demián! Muchas gracias, no me había percatado. ¿Cierro el issue?

**Comment by dna33** [comment]
No, mantengámoslo abierto. Con los edits que hicimos sirve para todes.

**Comment by elaval** [comment]
@dna33 sugiero actualizar el README incluyendo este producto (muchos nos informamos de los productos disponibles x esta vía)

Y ... sugiero incluir la palabra "Antígeno" (además de Ag) en la descripción del producto ya que este es el nombre que utiliza el Ministerio de Salud en el reporte diario y así se facilita la comunicación (y búsqueda)

Saludos y gracias

![image](https://user-images.githubusercontent.com/68602/121179408-b888f900-c82d-11eb-9679-4f4abd3022f7.png)

**Comment by dna33** [comment]
Hola Ernesto, el equipo lo incluyó  en el readme el mismo fin de semana, junto con el producto de test PCR (numero 7), lo que pasa es que lo metimos más arriba que la parte que leíste y tomaste la foto. De cualquier manera, nos parece una buena recomendación la tuya, vamos a incorporar “Antigeno (Ag)” como sugieres a la descripción.

![Screen Shot 2021-06-08 at 10 04 33 AM](https://user-images.githubusercontent.com/12438811/121199524-01967880-c841-11eb-9812-fa3be1bc7bc2.png)

**Comment by MinCiencia** [comment]
Hecho el cambio, gracias: https://github.com/MinCiencia/Datos-COVID19/commit/0b625a1f06226077ff9befe862a53b4b28153aa1
